source/Editor.js

```diff
@@ -1993,11 +1993,49 @@ var keyHandlers = {
             // If the commonAncestor was in a mathjax equation, delete the parent. Namely the mathjax equation.
             if (range.commonAncestorContainer && 
                 range.commonAncestorContainer.parentElement && 
-                checkInSVG.test(range.commonAncestorContainer.nodeName)
+                (checkInSVG.test(range.commonAncestorContainer.nodeName) || checkInSVG.test(range.commonAncestorContainer.parentElement.nodeName))
```

Comment:
This is trying to catch strange cases where text nodes are passed by the browser as commonAncestorContainer. The most common occurrence of this, is that the text node within a script tag will be passed as the commonAncestorContainer. This intends to confirm that we've covered all the bases, in terms of what the browser could send us.